README.adoc

```diff
@@ -34,6 +34,32 @@ If you have administrative access to the general `openshift` namespace and want
 oc replace -n openshift --force -f ...
 ----
 
+To import the image, or update an existing image:
+
+[source, bash]
+----
+oc import-image eap-cd-openshift:12
+----
+
+Optionally include namespace to the import:
+[source, bash]
+----
+oc -n myproject import-image eap-cd-openshift:12
+----
+
+Example:
```

Comment:
I don't see how what follows is an example of what came above. Is there some missing text?

Reply:
I rewrote that a few times. I'm trying to suggest that if you want to force the image to update, you can use the import-image, but you don't actually have to, as the first time you reference it, it will pull.

Maybe I should say 'To force import the image, or update ...'

Reply:
I updated this to try and clarify. I want to have the minimum we can here and link to the docs for anything else.